src/js/pages/settings/UISettingsPage.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { Trans } from "@lingui/macro";
+import { Link } from "react-router";
+
+import Breadcrumb from "#SRC/js/components/Breadcrumb";
+import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
+import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
+import ConfigurationMapHeading from "#SRC/js/components/ConfigurationMapHeading";
+import ConfigurationMapLabel from "#SRC/js/components/ConfigurationMapLabel";
+import ConfigurationMapRow from "#SRC/js/components/ConfigurationMapRow";
+import ConfigurationMapSection from "#SRC/js/components/ConfigurationMapSection";
+import ConfigurationMapValue from "#SRC/js/components/ConfigurationMapValue";
+import EventTypes from "#SRC/js/constants/EventTypes";
+import Languages from "#SRC/js/constants/Languages";
+import Loader from "#SRC/js/components/Loader";
+import Page from "#SRC/js/components/Page";
+import LanguageModalStore from "#SRC/js/stores/LanguageModalStore";
+import UserSettingsStore from "#SRC/js/stores/UserSettingsStore";
+import Util from "#SRC/js/utils/Util";
+import { SAVED_STATE_KEY } from "#SRC/js/constants/UserSettings";
+
+import LanguageModalActions from "#SRC/js/events/LanguageModalActions";
+
+const Loading = () => <Loader size="small" type="ballBeat" />;
+
+const UISettingsBreadcrumbs = () => {
+  const crumbs = [
+    <Breadcrumb key={-1} title="UI Settings">
```

Comment:
Do we translate this `title` as well? @brandonc

Reply:
Good catch - we probably should